shex/sio-model/disability.shex

```diff
@@ -0,0 +1,72 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX snomedct: <http://purl.bioontology.org/ontology/SNOMEDCT/>
+
+:personShape IRI {
+  a [ sio:SIO_000498 ] ;
+  sio:SIO_000228 @:roleShape ;
+  sio:SIO_000217 @:qualityShape
+}
+
+:roleShape IRI {
+  a [ obo:OBI_0000093 ] ;
+  sio:SIO_000356 @:testprocessShape
+}
+
+:qualityShape IRI {
+  a [ obo:NCIT_C21007 ] ;
+  sio:SIO_000642 @:answerShape ;
+  sio:SIO_000216 @:scoreShape
+}
+
+:questionnaireShape IRI {
+  a [ obo:NCIT_C130322 ] ;
+  sio:SIO_000028 @:questionnaire_questionShape
+}
+
+:questionnaire_questionShape IRI {
+  a [ obo:NCIT_C130916 ] ;
+  sio:SIO_000300 xsd:string
+}
+
+:testprocessShape IRI {
+  a [ obo:OMIT_0028217 ] ;
+  sio:SIO_000230 @:questionnaire_questionShape ;
+  sio:SIO_000229 @:answerShape
+}
+
+:answerShape IRI {
+  a [ sio:SIO_000340 ] ;
+  sio:SIO_000221 obo:UO_0000033 ;
+  sio:SIO_000300 xsd:string
```

Comment:
According to [example rdf ](https://github.com/NuriaQueralt/CDE-semantic-model/blob/develop/example-data/turtle/sio-model/disability.ttl)`xsd:string` should be `xsd:integer`